src/dev-spaces-preview/azext_dev_spaces_preview/azext_metadata.json

```diff
@@ -1,4 +1,4 @@
 {
-    "azext.minCliCoreVersion": "2.0.27",
+    "azext.minCliCoreVersion": "2.0.32",
```

Comment:
This PR relies on the CLI's next release right?
If so, change this to `2.0.33.dev0`